### PR TITLE
init for commands

### PR DESCRIPTION
**Pull request by SamYuan1990**
in this PR, add command download support as a step to get ready for integrate with kepler model server.

### CODE REVIEW

**Review by sunya-ch — APPROVED** [review]
/lgtm

File: index.js [hunk 1]
```diff
@@ -64,6 +64,19 @@ function installLibbpf(libbpf_version) {
...
+  executeCommand("mv ./kubectl /usr/local/bin/kubectl");
+  core.info(`Get kustomize`);
+  executeCommand("curl -s \"https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh\" | bash");
+  executeCommand("chmod +x kustomize");
+  executeCommand("mv kustomize /usr/local/bin/");
+}
+
+function installKind() {
+  core.info(`Get Kind with latest version`);
+  executeCommand("curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.22.0/kind-linux-amd64","fail to install kind");
```
Comment (vprashar2929) [inline index.js:77]: Do we want to make the Kind version fixed? Can we make it something like `installKind(kind_version)`?

**Review by vprashar2929 — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (SamYuan1990) [reply index.js:77]: let's make the version been fixed for now, we can improve in PR's next.

Comment (SamYuan1990) [reply index.js:77]: #107 openned.

**Pull request merged**